integrations/zendesk/src/actions/hitl.ts

```diff
@@ -0,0 +1,50 @@
+import { getZendeskClient } from '../client'
+import * as bp from '.botpress'
+
+export const startHitl: bp.IntegrationProps['actions']['startHitl'] = async ({ ctx, input, client }) => {
```

Comment:
HITLINT change number 2) bot creates conversation for the agent (agent not yet assigned at this point)